core/src/tangram.cpp

```diff
@@ -69,12 +69,8 @@ void initialize(const char* _scenePath) {
 
         loader.loadScene(sceneString, *m_scene, *m_tileManager, *m_view);
 
-        Primitives::setColor(0xffffff);
-        RenderState::configure();
     }
 
-    while (Error::hadGlError("Tangram::initialize()")) {}
```

Comment:
Yeah, as a matter of fact I'll add an error check in `setupGL()`. It's a useful thing to have, we just can't use `glGetError()` in `initialize()` because we may not have a GL context at that point.